File: cp_rust/src/bin/cses-1069-repetitions.rs
// Created by Ayush Biswas at 2025/07/09 22:16
// https://cses.fi/problemset/task/1069
use cp_lib::*;

// @code begin
use cpio::*;
use itertools::Itertools;

sol! {
    fn solution(
        a: [char]
    ) -> usize {
        a
            .into_iter()
            .group_by(|&ai| ai)
            .map(|g| g.len())
            .max()
            .unwrap()
    }
}

// @code end

File: cp_haskell/bin/cses-1069-repetitions.hs
-- Created by Ayush Biswas at 2026/03/03 15:43
-- https://cses.fi/problemset/task/1069

-- @code begin
import Data.List

main :: IO ()
main = do
  s <- getLine
  let g = maximum $ length <$> group s
  print g

-- @code end